libraries/src/Language/Language.php

```diff
@@ -193,6 +193,16 @@ public function __construct($lang = null, $debug = false)
 		$this->metadata = LanguageHelper::getMetadata($this->lang);
 		$this->setDebug($debug);
 
+		/*
+		Let's load the default override once, so we can profit from that, too
+		But make sure, that we don't enforce it on each language file load.
+		So don't put it in $this->override
```

Comment:
https://developer.joomla.org/coding-standards/inline-code-comments.html 

```
Comment blocks that introduce large sections of code and are more than 2 lines long should use /* */ (C) style and should use * on each line with the same space/tab rules as doc blocks.
```

```suggestion
		* Let's load the default override once, so we can profit from that, too
		* But make sure, that we don't enforce it on each language file load.
		* So don't put it in $this->override
```